app/src/main/java/org/simple/clinic/widgets/DividerItemDecorator.kt

```diff
@@ -5,25 +5,30 @@ import android.graphics.Canvas
 import android.graphics.Rect
 import android.graphics.drawable.Drawable
 import android.view.View
-import androidx.annotation.DrawableRes
-import androidx.core.content.ContextCompat
+import androidx.core.content.res.use
 import androidx.recyclerview.widget.RecyclerView
-import org.simple.clinic.R
+import timber.log.Timber
 import kotlin.math.roundToInt
 
+private val ATTRS = intArrayOf(android.R.attr.listDivider)
+
 class DividerItemDecorator(
     val context: Context,
     val marginStart: Int,
     val marginEnd: Int,
-    @DrawableRes val dividerDrawable: Int = R.drawable.divider
 ) : RecyclerView.ItemDecoration() {
 
   private val mBounds = Rect()
 
   private var divider: Drawable? = null
 
   init {
-    divider = ContextCompat.getDrawable(context, dividerDrawable)
+    context.obtainStyledAttributes(ATTRS).use {
+      divider = it.getDrawable(0)
+    }
+    if (divider == null) {
+      Timber.w("@android:attr/listDivider was not set in the theme used")
```

Comment:
Should we keep the timber statements?

Reply:
Yes, this is a warning log to indicate the theme level attribute is not set. Since this is not an important operation, we should not crash it. So we are showing a warning so that we can resolve this during the development cycle.